docs/src/main/asciidoc/resteasy-reactive-migration.adoc

```diff
@@ -104,9 +104,11 @@ RESTEasy Reactive does not support this feature. Instead, users that have provid
 using one of the methods described in the xref:cdi-reference.adoc#bean_discovery[Bean Discovery] section of the CDI guide.
 
 === Multipart support
-
 HTTP Multipart support in RESTEasy Reactive does **not** reuse the same types or annotations as RESTEasy Classic and thus users are encouraged to read <<resteasy-reactive#multipart,this>> part of the reference documentation.
 
+NOTE: Users migrating multipart resources to RESTEasy Reactive should be aware of the configuration parameter `quarkus.http.limits.max-form-attribute-size`, as this poses an upper limit to the size of each part.
```

Comment:
I see no reason to be repeating the information. I would drop this part and make the other part you added a note

Reply:
This behaviour and parameter is completely new to users migrating from Classic to Reactive. Shouldn't the migration page mention anything about this at all?

Reply:
Sorry, I had not noticed the doc that was changed. My bad